syntax = "proto3";

message FileObject {
    optional string status = 1;
    optional string uuid = 2;
    optional string version = 3;
    optional string content = 4;
    optional string name = 5;
}

service Serve {
    rpc Write(FileObject) returns (FileObject) {}
    rpc Read(FileObject) returns (FileObject) {}
    rpc Delete(FileObject) returns (FileObject) {}
}
